include/nektar_interface/function_basis_projection_alt.hpp

```diff
@@ -0,0 +1,179 @@
+#pragma once
```

Comment:
The rest of the code base is using include guards rather than pragma once (which is non-standard)
```suggestion
#ifndef _NESO_NEKTAR_INTERFACE_FUNCTION_BASIS_PROJECTION_ALT_HPP
#define _NESO_NEKTAR_INTERFACE_FUNCTION_BASIS_PROJECTION_ALT_HPP
```